lms/djangoapps/grades/tasks.py

```diff
@@ -23,7 +23,9 @@
 
 
 @task(default_retry_delay=30, routing_key=settings.RECALCULATE_GRADES_ROUTING_KEY)
-def recalculate_subsection_grade(user_id, course_id, usage_id, only_if_higher, raw_earned, raw_possible, **kwargs):
+def recalculate_subsection_grade(
```

Comment:
The model-level events are the most valuable events by far in my mind. They are the transactional events that allow construction of a student's grade history. Without those events I don't think it's worthwhile to have any eventing around grading at all.